starter_project_backend_nest/src/article/article.service.ts

```diff
@@ -50,30 +50,28 @@ export class ArticleService {
 
       if (newEntries.title) article.title = newEntries.title;
       if (newEntries.content) article.content = newEntries.content;
-      if (newEntries.author) {
-        if (newEntries.author.firstName)
-          article.author.name = newEntries.author.firstName;
-        if (newEntries.author.lastName)
-          article.author.lastName = newEntries.author.lastName;
-        if (newEntries.author.bio) article.author.name = newEntries.author.bio;
+      if (newEntries.authorUserId) {
```

Comment:
I think it's better to check the user exists by this id in the database before you insert the user.

Reply:
fixed